### PR TITLE
Better shotgun inhands

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Someone correctly pointed out to me that the other ones would need a free hand, and two hand component still isn't done yet. Whoever is working on it, get going for zero pay, and do it faster.

Since people did not like how THICC and BEEFY the inhands were as well, I made them a little shorter to make them look less like they're 7 ft long.

image
64xshottygun

This is what they're going to look like now, I'll bring back those two hand ones once the component is done 👀 

for combat shotgun, regular shotgun, riot shotgun, and double barrel.

## Changelog
:cl:
imageadd: New shotgun inhands
/:cl:

### CODE REVIEW

File: code/datums/looping_sounds/machinery_sounds.dm [hunk 1]
```diff
@@ -42,4 +42,4 @@
 	mid_sounds = list('sound/machines/microwave/microwave-mid1.ogg'=10, 'sound/machines/microwave/microwave-mid2.ogg'=1)
 	mid_length = 10
 	end_sound = 'sound/machines/microwave/microwave-end.ogg'
-	volume = 90
\ No newline at end of file
+	volume = 90
```
Comment (actioninja) [inline code/datums/looping_sounds/machinery_sounds.dm]: I don't know how this happened but alright

**Comment by Arathianen** [comment]
It looks amazing! Great job.

Is the firing animation changed at all?

File: code/datums/looping_sounds/machinery_sounds.dm (hunk 1 above)
Comment (actioninja) [reply code/datums/looping_sounds/machinery_sounds.dm]: if you're just looking at this and not the full changes, this isn't actually an addition it's a newline at end of file change

**Comment by actioninja** [comment]
no

**Comment by Tlaltecuhtli** [comment]
how does it look while wearing a shield or dual wielding?

**Comment by actioninja** [comment]
fucked up

**Comment by InsaneKatrina** [comment]
Man.

Those are some big guns

**Comment by Shapsy** [comment]
oh my god

**Comment by Tlaltecuhtli** [comment]
what the fuck 😂

**Comment by Dawson1917** [comment]
It's going to look very awkward being shot by someone who has their guns pointed at the ceiling

**Comment by MrFluffster** [comment]
_Soulless._

Absolutley amazing, perfect fit.

**Comment by Tlaltecuhtli** [comment]
what the fuck 😂

**Comment by Onule** [comment]
notfrickingaround

**Comment by nicbn** [comment]
That's hilariously awful

And I love it

**Comment by dr-bee** [comment]
5-6ft tall shotguns.

Those are basically punt guns.

**Comment by necromanceranne** [comment]
That looks so fucking stupid and I love it.

**Pull request merged**